packages/patternfly-3/patternfly-react-extensions/src/components/VerticalTabs/VerticalTabs.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import VerticalTabsTab from './VerticalTabsTab';
```

Comment:
Could this just be `VerticalTab`? Ie. `VerticalTabs` use `VerticalTab` components.

Reply:
Our convention has been to use the full component name, then the subcomponent name.